eclipse-collections/src/main/java/org/eclipse/collections/impl/block/procedure/MultimapEachPutProcedure.java

```diff
@@ -22,16 +21,9 @@
  */
 public final class MultimapEachPutProcedure<K, V> implements Procedure<V>
 {
-    private static final long serialVersionUID = 1L;
+    private static final long serialVersionUID = 2L;
     private final MutableMultimap<K, V> multimap;
     private final Function<? super V, ? extends Iterable<K>> keyFunction;
-    private final Procedure2<K, V> eachProcedure = new Procedure2<K, V>()
-    {
-        public void value(K key, V value)
-        {
-            MultimapEachPutProcedure.this.multimap.put(key, value);
-        }
-    };
 
```

Comment:
Removing this field shrinks the serialized form, but could be less efficient. I can drop this change from the patch.